ale_linters/rust/cargo.vim

```diff
@@ -3,19 +3,89 @@
 
 
 function! ale_linters#rust#cargo#GetCargoExecutable(bufnr) abort
-    if ale#util#FindNearestFile(a:bufnr, 'Cargo.toml') !=# ''
-        return 'cargo'
-    else
-        " if there is no Cargo.toml file, we don't use cargo even if it exists,
-        " so we return '', because executable('') apparently always fails
-        return ''
-    endif
+	if ale#util#FindNearestFile(a:bufnr, 'Cargo.toml') !=# ''
+		return 'cargo'
```

Comment:
Could you change your tab indents to four spaces, so it matches the rest of the codebase?